src/coq_rules.ml

```diff
@@ -90,11 +118,48 @@ let create_ccoq sctx ~dir =
   ; coqpp  = rr "coqpp"
   }
 
+(* compute include flags and mlpack rules *)
+let process_ml_deps ~scope ~stdlib_dir libs =
+
+  (* coqdep expects an mlpack file next to the sources otherwise it
+   * will omit the cmxs deps *)
+  let ml_pack_file lib =
+    let plugin_name = Option.value_exn (Result.ok_exn (Lib.main_module_name lib)) in
```

Comment:
Why is this `Option.value_exn`? Is it not possible to use unwrapped libs here?

Reply:
Not sure how this works, what I need to get is the path to the build-time `.cmxs`, what is the best way to do it?

Reply:
Use `Lib.plugins`. It will give you cma/cmxs based on bytecode/native.

Reply:
Done, seems to work well, thanks for the tip!